favoritesSearch.html

```diff
@@ -1,678 +1,101 @@
-	if (thumbs == null || thumbs.length == 0)
-		return;
-
-	// Get a random thumb
-	var n = thumbs.length;
-	var rand = Math.floor(Math.random() * n);	
-	var thumb = thumbs[rand];
-
-	// Get its id
-	var as = thumb.getElementsByTagName("a");
-	var id = new URLSearchParams(as[0].search).get("id");
-
-	// Set the href to the thumb's image
-	var random = document.getElementById("random");
-	random.href = "showOriginalRedirect.html?id=" + id;
-}
-
-// Deletes the "Show/Hide" content from the page
-function deleteShowHide(testDiv)
-{
-	var allChildren = testDiv.childNodes;
-	for (var i = 0; i < allChildren.length; i++)
-	{
-		if (0 != allChildren[i].childNodes.length && allChildren[i].childNodes[0].tagName == "B")
-		{			
-			allChildren[i].parentNode.removeChild(allChildren[i]);
-			return;
-		}
-	}
-}
-
-async function sleepAndRetry(offset, commands)
-{
-	await sleep(500);
-	sendNext(offset, commands);
-}
-
-// Stops all requests
-function stopItAll()
-{
-	window.stop();
-	document.getElementById("spinner").style.display = "none";
-}
-
-// Gets the value of a specific cookie
-function getCookie(name) {
-	const value = `; ${document.cookie}`;
-	const parts = value.split(`; ${name}=`);
-	if (parts.length === 2) return parts.pop().split(';').shift();
-	return "";
-}
-
-</script></body></html>
\ No newline at end of file
+	<head>
+		<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+  		<link rel="stylesheet" href="styles.css">
+		<script src="awesomplete.min.js" type="text/javascript" defer></script>
+		<script src="api.js" type="text/javascript" defer></script>
+		<script src="tag-logic.js" type="text/javascript" defer></script>
+		<script src="misc.js" type="text/javascript" defer></script>
+		<script src="loading.js" type="text/javascript" defer></script>
+		<script src="ui.js" type="text/javascript" defer></script>
+		<title>Rule34 Favorites Search</title>
+		<link rel="preconnect" href="https://fonts.googleapis.com/" crossorigin="true">
+		<link rel="preconnect" href="https://fonts.gstatic.com/">
+	</head>
+	<body id="body">
+		<div id="ui">
+			<form autocomplete="on" action="javascript:void(0);">
+				<input type="text" id="userId" placeholder="User Id" autocomplete="on">
+				<div class="awesomplete"><input id="tags" name="tags" size="30" type="text" value="" autocomplete="off" aria-autocomplete="list"><ul hidden=""></ul><span class="visually-hidden" role="status" aria-live="assertive" aria-relevant="additions"></span></div>
+				<input type="submit" id= "submit">
+				<a href="tutorial.html" target="_blank">About</a>
+			</form>
+			<input type="checkbox" id="advancedMode">
+			<label for="advancedMode" style="white-space:pre-wrap;">Advanced Mode (loads API data for images, allowing you to sort and filter them)</label>
+			<br>
+			<input type="checkbox" id="disableUI">
+			<label for="disableUI">Minimize UI</label>
```

Comment:
Can we make this checked by default? I think it's a little overwhelming to be presented with the big "advanced" section when first loading the the page.

Reply:
Agreed